grafana/README.md

```diff
@@ -0,0 +1,43 @@
+# Using Grafana with Linkerd
+
+You can install Grafana in various ways, like using the [Grafana official Helm
+chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana), or the
+[Grafana Operator](https://github.com/grafana-operator/grafana-operator). Hosted
+solutions are also available, like [Grafana
+Cloud](https://grafana.com/products/cloud/).
+
+The file `grafana/values.yaml` provides a default Helm config for the [Grafana
+official Helm
+chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana), which
+pulls the Linkerd dashboards published at
+<https://grafana.com/orgs/linkerd/dashboards>.
+
+You can install the chart like this:
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts helm install
+grafana -n grafana --create-namespace grafana/grafana \
+  -f https://github.com/linkerd/linkerd2/blob/main/grafana/values.yaml
```

Comment:
Shouldn't this be `https://raw.githubusercontent.com/linkerd/linkerd2/blob/main/grafana/values.yaml`?